lib/reports/pils/index.js

```diff
@@ -0,0 +1,44 @@
+const moment = require('moment');
+
+const formatDate = date => {
+  return date ? moment(date).format('YYYY-MM-DD') : '';
+};
+
+module.exports = ({ db }) => {
+
+  const query = () => {
+    return db.asl('pils')
+      .select('pils.*')
+      .select('establishments.id AS establishment')
+      .join('establishments', 'pils.establishment_id', '=', 'establishments.id')
```

Comment:
Why join establishments at all if we're only using the id?

Reply:
Did you want the name in the select instead?

Reply:
Because I copied it wholesale from the other PIL report and didn't bother to change this.

Reply:
I'm not really sure tbh. The establishment isn't mentioned in the ticket at all, but I can't imagine that it's _not_ going to be useful.

I might swap the name in.

Reply:
I did that.